.circleci/config.yml

```diff
@@ -1,7 +1,7 @@
 version: 2.1
 
 orbs:
-  android: wordpress-mobile/android@dev:publish-to-s3
+  android: wordpress-mobile/android@dev:should-publish-library-artifacts
```

Comment:
Up to you whether to wait for https://github.com/wordpress-mobile/circleci-orbs/pull/73 to be merged and update this before merging it, or point to the main orb in the parent PR  https://github.com/wordpress-mobile/WordPress-Utils-Android/pull/63

Reply:
I've updated this in 2852c5f9567e17cb9253e6682b95af6a6526eb8e. I'll merge it once CI is green 🤞